README.md

```diff
@@ -184,19 +184,19 @@ USAGE:
     simple [FLAGS] [OPTIONS]
 
 FLAGS:
-    -h, --help             Prints help information
-    -l, --list             Shows list of all possible Goose tasks and exits
-    -g, --log-level        Log level (-g, -gg, -ggg, etc.)
-        --manager          Enables manager mode
-        --no-hash-check    Ignore worker load test checksum
-        --no-stats         Don't print stats in the console
-        --only-summary     Only prints summary stats
-        --reset-stats      Resets statistics once hatching has been completed
-        --status-codes     Includes status code counts in console stats
-        --sticky-follow    User follows redirect of base_url with subsequent requests
-    -V, --version          Prints version information
-    -v, --verbose          Debug level (-v, -vv, -vvv, etc.)
-        --worker           Enables worker mode
+    -h, --help              Prints help information
+    -l, --list              Shows list of all possible Goose tasks and exits
+    -g, --log-level         Log level (-g, -gg, -ggg, etc.)
+        --manager           Enables manager mode
+        --no-hash-check     Ignore worker load test checksum
+        --no-reset-stats    Resets statistics once hatching has been completed
+        --no-stats          Don't print stats in the console
+        --only-summary      Only prints summary stats
```

Comment:
s/stats/statistics/

Reply:
Per above, this would introduce inconsistency -- if we want to expand `stats` to `statistics` we should open a new issue and do it everywhere.

Reply:
https://github.com/tag1consulting/goose/issues/131